src/atmosphere.ts

```diff
@@ -27,17 +27,13 @@ class Atmosphere {
   private numClouds = tweakables.cloud.num
 
   private display: Display
-  private clouds: Cloud[]
-  private activeSkies: ActiveSky[]
-  private sunnyCloudTextures: Texture2D[] // has a 1-1 relationship with DarkCloudTextures list
-  private darkCloudTextures: Texture2D[] // has a 1-1 relationship with SunnyCloudTextures list
+  private clouds = new Array<Cloud>()
+  private activeSkies = new Array<ActiveSky>()
+  private sunnyCloudTextures = new Array<Texture2D>() // has a 1-1 relationship with DarkCloudTextures list
+  private darkCloudTextures = new Array<Texture2D>() // has a 1-1 relationship with SunnyCloudTextures list
   private canvasManager: CanvasManager
 
   public constructor(display: Display, canvasManager: CanvasManager) {
-    this.activeSkies = []
```

Comment:
a common cleanup in this mr. we can init and have the type in one in the declaration above, just lets up remove a bunch of lines and have it init in one place